src/mscorlib/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1111,10 +1111,10 @@ internal struct Slot
 
     /// <summary>Padded head and tail indices, to avoid false sharing between producers and consumers.</summary>
     [DebuggerDisplay("Head = {Head}, Tail = {Tail}")]
-    [StructLayout(LayoutKind.Explicit, Size = 192)] // padding before/between/after fields based on typical cache line size of 64
+    [StructLayout(LayoutKind.Explicit, Size = 384)] // padding before/between/after fields based on worst case cache line size of 128
```

Comment:
It seems we may want to make this change ARM64 specific (put it under `#if ARM64`)